tests/test_lira_attack.py

```
@@ -28,6 +28,12 @@ def clean_up(name):
         os.remove(name)
 
 
+def test_parse_boolean_argument():
```

should we have a test to check the actual behaviour of the new options?

---

Added a test to do code coverage. Will add it later today to test the functionality of fail fast.